src/endpoints/sources.js

```diff
@@ -270,6 +270,21 @@ const putNarrative = contentTypesConsumed([
 ]);
 
 
+/* Group customizations
+ */
+
+/* GET
+ */
+const getGroupOverviewMarkdown = contentTypesProvided([
+  ["text/markdown", sendGroupOverview(req => req.context.source)],
+]);
+
+
+const getGroupOverview = contentTypesProvided([
+  ["text/markdown", getGroupOverviewMarkdown],
+]);
```

Comment:
Yeah, I largely followed the existing patterns for narrative endpoints since I imagined they'd be the most similar to group overview given Markdown type. 

So is the suggestion here to remove the `getGroupOverviewMarkdown` intermediate? I'm confused because
your other suggestions https://github.com/nextstrain/nextstrain.org/pull/570#discussion_r924794105 and https://github.com/nextstrain/nextstrain.org/pull/570#discussion_r924795051 seem to rely on this intermediate.

Reply:
Ah, suggestion is either to a) remove `getGroupOverviewMarkdown` or b) keep it but make the other two linked changes. So correct, not both. :-) I think I have a slight preference for (a), but it's not a big deal either way.

Reply:
Ok, removed `getGroupOverviewMarkdown`: 627831b36b9e35e7542b3d3f0ceee15270ec685b